.github/workflows/frontend-lighthouse.yml

```diff
@@ -0,0 +1,59 @@
+name: Frontend-Lighthouse
+
+on: 
+  issue_comment:
+    types: [ edited ]
+
+jobs:
+  capture_vercel_preview_url:
+    name: Capture Vercel preview URL
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: aaimio/vercel-preview-url-action@v1
```

Comment:
preffer web3 equivalent: https://fleek.co/

Reply:
ok, check in with Barry/Thore if any of these services cost $$

Reply:
Ack, I think we can rely on free tier plan at the moment. And also I left feature request to Fleek team. Hope we could migrate to it near future.